src/vde_backend/utils.py

```diff
@@ -5,7 +5,9 @@
 
 
 # standard library
+import json
 import os
+import os.path
 import math
 from datetime import timedelta
 
```

Comment:
main can be merge to this branch in order to update the logging mechanism in DEV mode. Git tells us that there will be a small conflict in the imports section.